recipes/cp2k-output-tools/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2022 Tiziano Müller
```

Comment:
Where did this license file come from? I don't see it in the upstream repo anywhere.

Reply:
Heya, yes I opened an issue about it: https://github.com/cp2k/cp2k-output-tools/issues/19, but do I need to wait on it being enacted?

Reply:
Do you need it to be in the release package (e.g. do I have to make a new release with the file present) or is it enough to be there? Besides, the package on pypi is already correctly list it as MIT since I've specified the license in the `pyproject.toml`.

Reply:
Confirmed that a license file is now present in https://github.com/cp2k/cp2k-output-tools